datahub-web-react/src/app/lineage/utils/getChildren.ts

```diff
@@ -30,12 +30,15 @@ export default function getChildren(entityAndType: EntityAndType, direction: Dir
     }
 
     if (direction === Direction.Downstream) {
-        if (entityAndType.type === EntityType.Dataset) {
+        if ('genericDownstreamLineage' in entityAndType.entity) {
             return (
-                entityAndType.entity.downstreamLineage?.downstreams.map((downstreams) => ({
-                    type: EntityType.Dataset,
-                    entity: downstreams.dataset,
-                })) || []
+                entityAndType.entity.genericDownstreamLineage?.entities?.map(
+                    (downstream) =>
+                        ({
+                            type: downstream?.entity?.type,
```

Comment:
Is the type variable even used in this case?

Reply:
yes, when we want to fetch that entity's children it is referenced